### PR TITLE
encapsulate path formation

### PR DESCRIPTION
**Pull request by lanej**

### CODE REVIEW

**Comment by lanej** [comment]
@technosophos @adamreese this is what i had in mind per our coversation in #355

File: util/structure.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package util
+
+import "path/filepath"
+
+// CachePath is the suffix for the cache.
+const CachePath = "cache"
+
+// CacheChartPath is the directory that contains a user's cached charts.
+const CacheChartPath = "cache/charts"
```
Comment (adamreese) [inline util/structure.go]: `CacheChartPath` can probably get axed.  We now use `cache/$repo`

**Comment by adamreese** [comment]
I like it!

**Comment by technosophos** [comment]
Yeah.

**Comment by technosophos** [comment]
Wow. The diff pretty much proves that you hit a valuable abstraction point.

**Pull request merged**